core/circuitbreaker/rule_manager.go

```diff
@@ -197,6 +198,10 @@ func onRuleUpdate(rules []*Rule) (err error) {
 		}
 		ruleSet = append(ruleSet, rule)
 		newBreakerRules[classification] = ruleSet
+
+		// update resource slot chain
+		misc.RegisterResourceRuleCheckSlot(rule.Resource, DefaultSlot)
```

Comment:
Do we need to unregister the slot if rule of resource delete?

Reply:
The slot is singleton and relate to resource.
If the resource is active, the slot chain is meaningful.
If the resource is inactive, the slot chain is not meaningful.
We should think of it case by case